style.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@500;600;700&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Oswald', sans-serif;
+}
+body{
+    min-height: 100vh;
+    background: #3d3d3d;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.container{
+    gap: 10px;
+    display: flex;
+
+    transform-style: preserve-3d;
+    transform: rotateY(30deg) rotateX(10deg);
+}
+.container .text{
+    position: relative;
+    width: 100px;
+    height: 100px;
+
+    transform-style: preserve-3d;
+    transition: 3.5s cubic-bezier(0,1.03,1,-0.08);
+    transition-delay: calc(0.25s * var(--j));
+}
+.container:hover .text{
+    transform: rotateX(-7290deg);
+}
+.container:hover .text:last-child{
+    transform: rotateX(630deg);
+}
+.container .text::before{
+    content: '';
+    position: absolute;
+
+    width: 100%;
+    height: 100%;
+    background: #242424;
+
+    transform-origin: left;
+    transform: rotateY(90deg) translateX(-50%);
+    transition: 3.5s ease-in;
+}
+.container:hover .text:nth-child(1):before, .container:hover .text:nth-child(2):before{
+    background: rgb(190, 71, 1);
+}
+.container:hover .text:nth-child(3):before{
+    background: rgb(151, 149, 149);
+}
+.container:hover .text:nth-child(4):before, .container:hover .text:nth-child(5):before{
+    background: rgb(1, 92, 1);
+}
+.container .text span{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    font-size: 4em;
+    color: #000;
+
+    background: #585858;
+    box-shadow: 2px 2px 20px 6px rgba(48, 47, 47, 0.7) inset;
+    transition: 3.5s ease-in;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    transform-style: preserve-3d;
+    transform: rotateX(calc(90deg * var(--i))) translateZ(50px);
+    text-shadow: -8px 8px 5px rgba(0, 0, 0, 0.4);
+}
+.container:hover .text:nth-child(1) span, .container:hover .text:nth-child(2) span{
+    color: #fff;
+    background: rgb(255, 153, 51);
+    box-shadow: 2px 2px 20px 6px rgba(187, 98, 9, 0.7) inset;
+}
+.container:hover .text:nth-child(3) span{
+    color: #000080;
+    background: white;
+    box-shadow: 2px 2px 20px 6px rgba(173, 173, 173, 0.7) inset;
+}
+.container:hover .text:nth-child(4) span, .container:hover .text:nth-child(5) span{
+    color: #fff;
+    background: rgb(19, 136, 8);
+    box-shadow: 2px 2px 20px 6px rgba(15, 97, 8, 0.7) inset;
+}
+
+@media screen and (max-width:400px){
```

Comment:
if the max-width is 600px then it would be more visible to users with a screen width below 600px.

Reply:
okk I'll change it